.coveragerc

```diff
@@ -3,3 +3,10 @@ omit =
     # we omit numba code since we cannot measure it
     ngmix/*_nb.py
     ngmix/fastexp.py
+    ngmix/leastsqbound.py
```

Comment:
This module is important for like every fit in ngmix. Do we really want to ignore it?

Reply:
Its hard to test most of the stuff in there.  Its all branches for various ways the fitter can behave.  I didn't know how to trigger them.

Reply:
I think coverage was only about 30%

Reply:
I think I'll put it back in to remind us to divise a way to test it.

But I don't have ideas